### ISSUE TITLE
Parsing of 1/mile is not possible

### ISSUE THREAD

**Issue by acimail01** [unitsofmeasurement/indriya#413]
```
SimpleUnitFormat.getInstance().alias(USCustomary.MILE, "mile");
Quantity<?> q1 = ServiceProvider.current().getFormatService().getQuantityFormat().parse("3 km");
Quantity<?> q2 = ServiceProvider.current().getFormatService().getQuantityFormat().parse("3 mi");
Quantity<?> q3 = ServiceProvider.current().getFormatService().getQuantityFormat().parse("3 mile");
// Quantity<?> q4 = ServiceProvider.current().getFormatService().getQuantityFormat().parse("3 1/km"); // Exception
Quantity<?> q5 = ServiceProvider.current().getFormatService().getQuantityFormat("EBNF").parse("3 1/km");
// Quantity<?> q6 = ServiceProvider.current().getFormatService().getQuantityFormat("EBNF").parse("3 1/mi"); // Exception
// Quantity<?> q7 = ServiceProvider.current().getFormatService().getQuantityFormat("EBNF").parse("3 1/mile"); // Exception
```

Parsing of 1/km with Simple-Parser is not possible
Parsing of 1/mile wirh EBNF-Parser is not possible

**Comment by keilw** [comment]
Could you please elaborate the use case for this?

Even in the [UCUM Specification](https://ucum.org/ucum) which is by far the most extensive catalog of units there is no definition of "1/km", "1/kg" or similar, meaning it would likely fail against `UCUMFormat` or UCUM's own test suites as well.

**Comment by keilw** [comment]
Fixed by #429, see [Common QuantityFormatDemo](https://github.com/unitsofmeasurement/uom-demos/blob/main/console/systems/common/src/main/java/tech/uom/demo/systems/common/QuantityFormatDemo.java )

Only `q4` fails with `SimpleUnitFormat`, that is a known problem. We may address that in a future version, but `EBNFUnitFormat` will work, once it can be enriched by the relevant systems. The demo shows, how you can do that in your own code already.
With `EBNFUnitFormat.getNewInstance()` you can create an isolated local instance and apply aliases of labels that do not affect the shared standard instance used by the service.
